src/components/essential/DlTextArea/DlTextArea.vue

```diff
@@ -113,9 +167,37 @@ export default defineComponent({
         warning: {
             type: Boolean,
             default: false
+        },
+        required: {
+            type: Boolean,
+            default: false
+        },
+        tooltip: {
+            type: String,
+            default: ''
+        },
+        optional: {
+            type: Boolean,
+            default: false
+        },
+        title: {
+            type: String,
+            default: ''
+        },
+        topMessage: {
+            type: String,
+            default: ''
+        },
+        hideClearButton: {
+            type: Boolean,
+            default: false
+        },
+        clearButtonTooltip: {
```

Comment:
this is a string. check how it behaves in dl-input today

Reply:
![image](https://github.com/dataloop-ai/components/assets/45905083/9779c332-828b-4e03-b0a3-a0dd909ce5ad)

Reply:
Opening a new issue about this. thanks!